sdcm/provision/scylla_yaml/auxiliaries.py

```diff
@@ -56,7 +56,7 @@ class SeedProvider(BaseModel):
     ]
     parameters: List[dict] = None
 
-    @validator("class_name", pre=True, always=True)
+    @field_validator("class_name", mode='before')
```

Comment:
This seems to be independent of python 3.13 change. I would extract it to a separate PR

Reply:
yes, but all those warning is in my way to the warning related to python 3.13

so they would stay here, until I'm done, and then it I'll put them in separate PRs